CDTDatastoreTests/Encryption/TD_DatabaseEncryptionTests.m

```diff
@@ -17,12 +17,13 @@
 
 #import "CDTEncryptionKeyNilProvider.h"
 #import "CDTHelperFixedKeyProvider.h"
+#import "CloudantSyncTests.h"
 #import "CloudantTests+EncryptionTests.h"
 #import "FMDatabase+SQLCipher.h"
 
 #import "TD_Database.h"
 
-@interface TD_DatabaseEncryptionTests : CloudantTests
+@interface TD_DatabaseEncryptionTests : CloudantSyncTests
```

Comment:
I'll raise an issue for it, because it might be worth merging those two into one.